css/component.css

```diff
@@ -0,0 +1,240 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+}
+
+:root {
+  --primary-color: #74777b;
+  --background-color: #e7ecea;
+}
+
+html {
+  font-size: 10px;
+  font-family: "Montserrat", Arial, Helvetica, sans-serif;
+}
+
+body {
+  background-color: var(--background-color);
+}
+
+.container {
+  display: flex;
+  /* flex-wrap: wrap; */
+  /* margin: 0; */
+}
+
+section {
+  border-bottom: 1px solid var(--default);
+  padding: 2.5rem 0;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  padding: 2rem;
+  border: 1px solid var(--primary-color);
+  position: sticky;
+  top: 0;
+  z-index: 100;
```

Comment:
Code is clean and the naming convention is followed well.
I just think this z-index is higher (which is not an issue), It just seems a little vague value. I hope this helps!

Reply:
I've made the change. Thank you for the review.